idea-plugin/build.gradle

```diff
@@ -73,8 +74,8 @@ tasks.withType(org.jetbrains.intellij.tasks.PrepareSandboxTask) {
     dependsOn configurations.runtimeClasspath
 
     // Also pack the formatter in its own directory
-    into("${pluginName}/impl") {
```

Comment:
This variable was not resolved, so the current version was no longer included in our plugin!

Reply:
Nice find! Interesting that this worked before!

Reply:
I'm not entirely sure when it broke since it only failed when the idea plugin used a newer version than the repo